roles/common/handlers/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: Start GlusterFS
+  service: name=glusterfs-server state=started enabled=true
```

Comment:
Handlers is for running tasks when something changes, and the main use case is for restarting services when a setting changes. So I suggest this is moved to the GlusterFS role and changed to restart the server